docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,56 @@
+# Security and Permissions
+
+## Comparison table
+
+Refer to this table for a overview of the permissions needed for the different ways to run Renovate:
+
+| Permission name   | Renovate hosted app |  Forking Renovate  |
+| ----------------- | :-----------------: | :----------------: |
+| Dependabot alerts |       `read`        |   not applicable   |
+| Administration    |       `read`        |       `read`       |
+| Metadata          |       `read`        |       `read`       |
+| Checks            | `read` and `write`  |   not applicable   |
+| Code              | `read` and `write`  |       `read`       |
+| Commit statuses   | `read` and `write`  | `read` and `write` |
+| Issues            | `read` and `write`  | `read` and `write` |
+| Pull Requests     | `read` and `write`  | `read` and `write` |
+| Workflows         | `read` and `write`  |   not applicable   |
+| Deployments       |   not applicable    |       `read`       |
```

Comment:
It's a bit odd that Forking Renovate needs `read` access to `deployments` while the full fat hosted Renovate app does not.

Reply:
Maybe we can have a common place to document why/how each permission is used. Can we do it in the first table to avoid duplication?

Reply:
> Can we do it in the first table to avoid duplication?

I've put a _why_ column in the table. 😉 

> It's a bit odd that Forking Renovate needs `read` access to `deployments` while the full fat hosted Renovate app does not.

Do you want to make the permissions equal for both apps?